src/MiniProfiler.Shared/Timing.cs

```diff
@@ -77,28 +78,76 @@ public Timing(MiniProfiler profiler, Timing parent, string name, decimal? minSav
         /// Gets or sets All sub-steps that occur within this Timing step. Add new children through <see cref="AddChild"/>
         /// </summary>
         [DataMember(Order = 5)]
-        public List<Timing> Children { get; set; }
+        public List<Timing> Children
```

Comment:
This approach (like earlier comments) creates *a lot* of allocations. A new list per accessor really just isn't tenable from a performance or goals standpoint for the library. I'm just not sure any of the proposed solutions is lightweight enough to work.